### PR TITLE
support node 14

### PR DESCRIPTION
**Pull request by jsaur**
Signed-off-by: Jacob Saur <jsaur@kiva.org>

### CODE REVIEW

File: src/router/agent.router.module.ts [hunk 1]
```diff
@@ -23,14 +23,15 @@ export class AgentRouterModule implements NestModule {
      * TODO figure out how to handle case where there is no agent - queue their messages?
      */
     configure(consumer: MiddlewareConsumer) {
-        consumer.apply(createProxyMiddleware('/v1/router',
+        const middleware: any = createProxyMiddleware('/v1/router',
```
Comment (matt-raffel-kiva) [inline src/router/agent.router.module.ts:26]: is this change required because of node + package version changes or just refactor?

**Review by matt-raffel-kiva — APPROVED** [review]

File: src/router/agent.router.module.ts (hunk 1 above)
Comment (jsaur) [reply src/router/agent.router.module.ts:26]: Ya for some reason on the latest version of nestjs it didn't like the type, but things work fine without it
